fintrack/users/serializer.py

```diff
@@ -6,13 +6,14 @@
 class UserSerializer(serializers.ModelSerializer):
     class Meta:
         model = User
-        fields = ('username', 'email', 'password', 'address')
+        fields = ('username', 'email', 'password', 'address', 'is_staff')
 
     def create(self, validated_data):
         user = User(
             username=validated_data['username'],
             email=validated_data['email'],
-            address=validated_data['address']
+            address=validated_data['address'],
+            is_staff=validated_data['is_staff']
```

Comment:
why are we creating a staff user, most of the time this is used to control the Django admin panel access, like who can add, view, edit, delete objects